src/core.ml

```diff
@@ -474,7 +474,7 @@ module MakeExpert
         aux s.slake_vars env
 
   let _13__check_reason_when_unsat _env =
-    Logs.app ~src (fun p -> 
+    Logs.info ~src (fun p ->
```

Comment:
This is printed only when checking invariants, which is an option that is only activated when debugging. In that precise case, we do want to print stuff.
Actually, the error message is so bad I think we should just remove this `_13_` function.

Reply:
Still, we shouldn't print at the `App` level. Only binaries can print at `App` level. We can replace `Logs.info` by `Logs.debug` instead. But `Logs.app` is wrong because if you use the library and activate the debug flag, you will get these messages on your `App` level... The correctness of the error is another problem ;)